snuba/admin/views.py

```diff
@@ -571,6 +531,56 @@ def clickhouse_trace_query() -> Response:
         )
 
 
+def gather_profile_events(query_trace: TraceOutput, storage: str) -> None:
```

Comment:
this file is pretty heavily overloaded, can you move this into its own file?

Reply:
Yes. I will refactor the changes related to gather_profile_events in a separate follow-up PR